scripts/install

```diff
@@ -302,15 +305,15 @@ ynh_system_reload --service_name=dnsmasq --action=restart
 # INSTALL THE CRON JOB
 #=================================================
 
-cp $pihole_local_repo/advanced/pihole.cron /etc/cron.d/pihole
+cp $pihole_local_repo/advanced/Templates/pihole.cron /etc/cron.d/pihole
 # Remove git usage for version. Which fails because we use here a release instead of master.
 ynh_replace_string ".*updatechecker.*" "#&" /etc/cron.d/pihole
 
 #=================================================
 # BUILD THE LISTS WITH GRAVITY
 #=================================================
 
-cp "$pihole_local_repo/adlists.default" "$pihole_storage/adlists.default"
+cp "../conf/adlists.default" "$pihole_storage/adlists.default"
```

Comment:
Why using a local copy of this list ?

Reply:
is not on new install i dont see is for fix error

Reply:
and is better for add your list